lib/ramble/ramble/pipeline.py

```diff
@@ -572,6 +572,7 @@ def _execute(self):
             app_inst.add_expand_vars(self.workspace)
             exec_str = app_inst.expander.expand_var(self.executor)
             exec_parts = shlex.split(exec_str)
+
```

Comment:
nit; Can you remove this change?

Reply:
oops, sorry about that, leftover from an interactive debugging change! Removed.